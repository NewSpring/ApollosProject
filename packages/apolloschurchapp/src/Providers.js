import React from 'react';
import { Providers } from '@apollosproject/ui-kit';
import ClientProvider from './client';

import theme from './theme'; // where theme looks like:

const NewSpringProviders = (props) => (
  <Providers themeInput={theme}>
    <ClientProvider {...props} />
  </Providers>
);

export default NewSpringProviders;
