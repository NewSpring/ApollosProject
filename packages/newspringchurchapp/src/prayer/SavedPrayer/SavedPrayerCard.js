import React from 'react';
import { BodyText, Card, CardContent } from '@apollosproject/ui-kit';

const SavedPrayerCard = () => (
  <Card>
    <CardContent>
      <BodyText>This is a saved prayer request</BodyText>
    </CardContent>
  </Card>
);

export default SavedPrayerCard;
