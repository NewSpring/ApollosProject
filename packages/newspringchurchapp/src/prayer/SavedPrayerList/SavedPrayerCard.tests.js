import React from 'react';
import renderer from 'react-test-renderer';

import Providers from 'newspringchurchapp/src/Providers';

import SavedPrayerCard from './SavedPrayerCard';

describe('SavedPrayerCard', () => {
  it('renders a saved prayer', () => {
    const tree = renderer.create(
      <Providers>
        <SavedPrayerCard />
      </Providers>
    );
    expect(tree).toMatchSnapshot();
  });
});
