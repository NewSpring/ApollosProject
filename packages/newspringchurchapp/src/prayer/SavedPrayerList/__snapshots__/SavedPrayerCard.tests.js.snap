// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SavedPrayerCard renders a saved prayer 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  onLayout={[Function]}
  style={
    Object {
      "bottom": 0,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <View
    onLayout={[Function]}
    style={
      Object {
        "flex": 1,
      }
    }
  />
  <View
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 16,
          "marginHorizontal": 16,
          "marginVertical": 12,
          "shadowColor": "rgba(133, 133, 133, 0.30000000000000004)",
          "shadowOffset": Object {
            "height": 1,
            "width": 0,
          },
          "shadowOpacity": 1,
          "shadowRadius": 6,
        }
      }
    >
      <View
        style={
          Object {
            "aspectRatio": undefined,
            "borderRadius": 16,
            "overflow": "hidden",
          }
        }
      >
        <View
          style={
            Object {
              "paddingHorizontal": 16,
              "paddingVertical": 16,
            }
          }
        >
          <Text
            bold={false}
            italic={false}
            style={
              Object {
                "color": "#303030",
                "fontFamily": "Colfax-Regular",
                "fontSize": 16,
                "lineHeight": 26,
              }
            }
          >
            This is a saved prayer request
          </Text>
        </View>
      </View>
    </View>
  </View>
</RCTSafeAreaView>
`;
